FND/Accounting/ISO4217-CurrencyCodes.rdf

```diff
@@ -2397,12 +2402,12 @@
 	
 	<owl:NamedIndividual rdf:about="&fibo-fnd-acc-4217;Paanga">
 		<rdf:type rdf:resource="&fibo-fnd-acc-cur;Currency"/>
-		<rdfs:label>Pa&apos;anga</rdfs:label>
-		<skos:definition>the currency Pa&apos;anga</skos:definition>
+		<rdfs:label>Pa’anga</rdfs:label>
+		<skos:definition>the currency Pa’anga</skos:definition>
```

Comment:
’ throws an error in hygiene test testHygiene0114.sparql - use &apos; instead.

Reply:
This is not a "smart quote" copied in but a genuine UTF-8 character and not used in a URI or XML element name. It does not fail the very fussy Ogygen XML character tests. We should doublecheck whether the hygiene test is too strict.

Reply:
@rivettp That only works if you also use the xml lang encoding - if it thinks it is a plain string then the encoding must be according to the rules in RDF and not using a special character.  So if you want it to work without the xml lang, you need to use a straight quote as Pawel suggests.

Reply:
@rivettp For Tonga, the xml language tag should be "xml:lang="to"

Reply:
@rivettp The proper encoding for the glottal stop, rather than a smart quote as the ISO published XML/spreadsheet included, is &#x02BB;

Reply:
Updated with escaped glottal stop 02BB as per the Unicode CLDR, agreed with Elisa. Checked the new file also loads and displays correctly in Protege